packages/tasks_panel/package.json

```diff
@@ -108,9 +108,15 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "VSCode Tasks Panel",
+      "title": "Task Explorer",
       "properties": {
-        "VSCode Tasks Panel.loggingLevel": {
+        "Task Explorer.enableSapWebAnalytics": {
```

Comment:
here should be package name according to documentation

Reply:
updated to package name